### PR TITLE
Handle back on task steps

### PR DESCRIPTION
**Pull request by pandafulmanda**
it works yay

### CODE REVIEW

File: src/components/task/index.cjsx [hunk 1]
```diff
@@ -32,24 +32,31 @@ module.exports = React.createClass
...
+  setStepKey: ->
     {stepIndex} = @context.router.getCurrentParams()
     # url is 1 based so it matches the breadcrumb button numbers
     crumbKey = if stepIndex then parseInt(stepIndex) - 1 else @getDefaultCurrentStep()
+    @setState({currentStep: crumbKey})
+
+  getInitialState: ->
     {
-      currentStep: crumbKey
+      currentStep: 0
```
Comment (philschatz) [inline src/components/task/index.cjsx]: Does this still work with the student dashboard sending the student directly to a specific step?

Comment (pandafulmanda) [inline src/components/task/index.cjsx]: yes, it's being set in componentWillMount

Comment (philschatz) [inline src/components/task/index.cjsx]: thanks!

**Comment by philschatz** [comment]
Does the URL update refetch the step?

**Comment by pandafulmanda** [comment]
you mean on back?  it doesn't

**Pull request merged**